src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,24 @@
+import classNames from 'classnames';
+import { Link, useLocation, useResolvedPath } from 'react-router-dom';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const location = useLocation();
+  const parentPath = useResolvedPath('../').pathname;
+
+  return (
+    <Link
+      to={{
+        pathname: parentPath + person.slug,
```

Comment:
The pathname is being concatenated with person.slug using '+'. It would be better to use a template literal here for better readability.